src/idiomatic/foundations-api-design/predictable-api/naming-conventions/09-owned.md

```diff
@@ -0,0 +1,21 @@
+---
+minutes: 2
+---
+
+# Owned
+
+Suffix for functions and methods about owned values in a context where
+references are common.
+
+```
+String::to_owned
+Cow::into_owned
+```
+
+<details>
+- Most often seen in to_owned methods, especially on strings.
+
+Some exceptions, such as Cow (copy on write) having an `into_owned` method that
```

Comment:
Why is it an exception?

(You can simply delete this part if you turn this into a question for the audience.)